ingest-app/src/cmr/ingest/config.clj

```diff
@@ -5,6 +5,11 @@
             [cmr.oracle.connection :as conn]
             [cmr.message-queue.config :as rmq-conf]))
 
+(defconfig ingest-accept-umm-version
```

Comment:
Just want to point out that the default value and the actual value are two different things.
I think we need to set the default value to be always the current-version so that we can run/build our local tests without having the environment value set. 
In the fields, we will set the environment values differently, depending on if it's development environment or operation environment. I need to change the doc string.